README.md

```diff
@@ -2182,6 +2187,85 @@ in [our JavaScript](https://github.com/codeforamerica/form-flow/blob/9a1c6f7a253
 
 We send the whole url to Mixpanel, including any query parameters, such as `lang`.
 
+## Interceptors
```

Comment:
We need a separate section about landmark pages, independent of this section.  This would be where we explain what landmark pages are intended for and catalog the ones that exist (right now, only the firstScreen, but I anticipate more soon).